### PR TITLE
fix n_state assertion in draw_samples

### PR DESCRIPTION
**Pull request by cxhernandez**
re https://github.com/msmbuilder/msmbuilder/issues/637

### CODE REVIEW

**Comment by rmcgibbo** [comment]
can you see if you can make the docstring more intelligible too?

**Comment by cxhernandez** [comment]
sure!

**Comment by cxhernandez** [comment]
Actually I don't think the docstring is all that bad. The error just had me confused.

**Comment by cxhernandez** [comment]
I made the docstring a little less verbose and added a check for whether the input is just a single list

**Comment by mpharrigan** [comment]
feel free to add something to docs/changelog.rst if you want :)

File: msmbuilder/msm/core.py [hunk 1]
```diff
@@ -165,39 +165,41 @@ def sample_discrete(self, state=None, n_steps=100, random_state=None):
...
-        sequences : list
-            List of state label sequences, each of which
-            has shape (n_samples_i), where n_samples_i is the length of
-            the ith trajectory.
+        sequences : list or list of lists
+            A sequence or list of sequences, in which each element corresponds
+            to a state label.
         n_samples : int
-            How many samples to return from each state
+            How many samples to return for any givem state.
```
Comment (kyleabeauchamp) [inline msmbuilder/msm/core.py]: given

Comment (cxhernandez) [inline msmbuilder/msm/core.py]: Thanks!

**Comment by cxhernandez** [comment]
I think the py3.4 test on Travis failed on something completely different.

**Pull request merged**